cypress/e2e/tests/pages/manager/cluster-provisioning-azure-rke2.spec.ts

```diff
@@ -0,0 +1,122 @@
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+import ClusterManagerListPagePo from '@/cypress/e2e/po/pages/cluster-manager/cluster-manager-list.po';
+import ClusterManagerCreateRke2AzurePagePo from '@/cypress/e2e/po/edit/provisioning.cattle.io.cluster/create/cluster-create-rke2-azure.po';
+import ClusterManagerDetailRke2AzurePagePo from '@/cypress/e2e/po/detail/provisioning.cattle.io.cluster/cluster-detail-rke2-azure.po';
+import PromptRemove from '@/cypress/e2e/po/prompts/promptRemove.po';
+import LoadingPo from '@/cypress/e2e/po/components/loading.po';
+
+// will only run this in jenkins pipeline where cloud credentials are stored
+describe('Provision Node driver RKE2 cluster with Azure', { testIsolation: 'off', tags: ['@manager', '@adminUser', '@jenkins'] }, () => {
```

Comment:
The structure _looks good to me_ in general in this PR. I'd just wait for feedback on js idioms and best practices from dev.

These provisioning tests should include the standard user role in the future. Could you add this to the backlog?  👍🏽